app/user-data-download/server/cronProcessDownloads.js

```diff
@@ -319,7 +333,7 @@ const sendEmail = function(userId) {
 	});
 };
 
-const makeZipFile = function(exportOperation) {
+const makeZipFile = async function(exportOperation) {
```

Comment:
if following below suggestion, the `async` is not needed anymore.

```suggestion
const makeZipFile = function(exportOperation) {
```